ocf_datapipes/transform/xarray/convert_satellite_to_int.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import xarray as xr
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+
+@functional_datapipe("convert_satellite_to_int8")
+class ConvertSatelliteToInt8IterDataPipe(IterDataPipe):
+    def __init__(self, source_dp: IterDataPipe):
+        super().__init__()
+        self.source_dp = source_dp
+
+    def __iter__(self):
+        for xr_dataset in self.source_dp:
+            xr_dataset = xr_dataset.clip(min=0, max=1023)
```

Comment:
hard coded value? could be moved to constants?

Reply:
For this, its specifically to int8, so 1023 is the max value it can take, it wouldn't make sense to set it to something else, so I think it should be fine here.